js/game.js

```diff
@@ -108,6 +108,7 @@ Game.displayChunk = function(key){
 
 Game.removeChunk = function(chunkID){
     Game.maps[chunkID].destroy();
+    Game.scene.cache.tilemap.remove(`chunk${chunkID}`);
```

Comment:
This is really needed because without it Phaser doesn't load again previously removed chunks.